packages/react/src/QuestionnaireForm/QuestionnaireForm.tsx

```diff
@@ -604,23 +605,48 @@ export function isQuestionEnabled(
   if (!item.enableWhen) {
     return true;
   }
+
   const enableBehavior = item.enableBehavior ?? 'any';
+
   for (const enableWhen of item.enableWhen) {
-    const expectedAnswer = getTypedPropertyValue(
-      {
-        type: 'QuestionnaireItemEnableWhen',
-        value: enableWhen,
-      },
-      'answer[x]'
-    );
     const actualAnswer = getTypedPropertyValue(
       {
         type: 'QuestionnaireResponseItemAnswer',
         value: answers[enableWhen.question as string],
       },
       'value[x]'
-    );
-    const match = deepEquals(expectedAnswer, actualAnswer);
+    ) as TypedValue | undefined; // possibly undefined when question unanswered
+
+    const expectedAnswer = getTypedPropertyValue(
+      {
+        type: 'QuestionnaireItemEnableWhen',
+        value: enableWhen,
+      },
+      'answer[x]'
+    ) as TypedValue;
+
+    let match: boolean;
+
+    const { operator } = enableWhen;
+    // We handle exists separately since its so different in terms of comparisons than the other mathematical operators
+    if (operator === 'exists') {
+      // if actualAnswer is not undefined, then exists: true passes
+      // if actualAnswer is undefined, then exists: false passes
+      match = !!actualAnswer === expectedAnswer.value;
+    } else if (actualAnswer === undefined) {
+      match = false;
+    } else {
+      const [{ value }] = evalFhirPathTyped(
+        `%actualAnswer ${operator?.replace('=', '~')} %expectedAnswer`,
```

Comment:
TIL the `~` operator in FHIRPath

Reply:
Yeah Matt was super helpful on that. He noticed that the `=` operator for `enableWhen` should actually be `~` in FHIRPath. Funny.